templates/vue/src/components/modals/NodeModal/index.vue

```diff
@@ -930,6 +930,67 @@ export default {
         if (!validPreviousAnswers) {
           errMsgs.push("Please select a previous activity to display")
         }
+        const questionsWithDragDropEnabled = this.node.typeData.activity.questions.filter(
```

Comment:
Can we group these somehow? Maybe move all of this validation into its own method? This drag and drop validation seems super long!